odoo/addons/base/models/ir_cron.py

```diff
@@ -205,6 +210,14 @@ def _acquire_one_job(cls, cr, job_ids):
         # been processed, its nextcall is updated to a date in the
         # future and the optionnal trigger is removed.
         #
+        # It is important to limit the query to a single job, it should
+        # ``where id=x``, not ``where id in (x, y, z, ...) limit 1``.
+        # In case the latter is executed, postgres snapshots the rows
+        # present in the id list in the current transaction. Another
+        # thread could acquire and update (e.g. the ``nextcall`` value)
+        # of one of those other crons and cause a silly serialisation
+        # error here even if we actually don't care of the other crons.
```

Comment:
```suggestion
        # It is important to limit the query to a single job, it should do
        # ``WHERE id = ...`` instead of ``WHERE id IN (...) LIMIT 1``.
        # In case the latter is executed, Postgres snapshots all the rows
        # present in the id list in the current transaction.  Another thread
        # could then acquire and update (the ``nextcall`` value) one of those
        # other jobs and cause a silly serialization error here even if we
        # actually should not depend on other jobs.
```